userguide/intro.rst

```diff
@@ -113,14 +119,37 @@ fixes.
 
 **Deprecated and Removed Features**
 
+* Upgrading from legacy (9.2 or earlier) versions of FreeNAS has been
+  removed from the installer.
+
+* GELI encryption has been deprecated and replaced by native ZFS
+  encryption. The |web-ui| remains backwards compatible with GELI
+  encryption keys and passphrases.
+
 * The *Legacy* replication method has been removed from
   :ref:`Advanced Replication Creation`.
 
 **New or Updated Software**
 
+* The OS has been updated to FreeBSD 12.
+
+* :literal:`avahi` has replaced :literal:`mdnsresponder`.
+
+* :literal:`collectd` is updated to version 5.9.
+
+* :literal:`Samba` has been updated to version 4.11.2.
+
+* :literal:`Sentry` has replaced the :literal:`Raven` Python module.
+
+* TLS 1.3 support has been added.
```

Comment:
Do we want to make mention of the ability to disable older unsupported TLS versions? I believe that's what NAS-103357 addressed.

Reply:
Not in the new/updated software section, but I will add a note about the new field in the general notes.